src/index.js

```diff
@@ -47,12 +47,13 @@ function handleExtendsNodes(tree, options, messages) {
         locals = JSON.parse(extendsNode.attrs.locals);
       } catch {}
     }
-    options.expressions.locals = merge(options.expressions.locals, locals);
-    options.plugins.push(expressions(options.expressions));
+
+    var options_expressions = merge(options.expressions, {locals});
```

Comment:
1. Not very clear about the purpose of just such a merger?
This will cost more than the current merger
2. var ?

Reply:
the merge is really suboptimal, and after testing it was proven unecessary. the new variable is not needed.